.github/workflows/config/spelling_allowlist.txt

```diff
@@ -10,9 +10,11 @@ CPTP
 CPU
 CPUs
 CUDA
+Composability
```

Comment:
I think you can probably lowercase it in the allow list and the spellcheck should be smart enough to recognize capitalization at the beginning of a sentence.

Reply:
Thanks. Updated in [a41da94](https://github.com/NVIDIA/cuda-quantum/pull/1040/commits/a41da9475b844ed6fca05fd907469c41bb65ffa4).